ci/infra/testrunner/platforms/terraform.py

```diff
@@ -2,143 +2,65 @@
-        return False
-
-    def cleanup(self):
-        """ Clean up """
-        try:
-            self._cleanup_platform()
-        except Exception as ex:
-            cleanup_failure = True
-            logger.warning("Received the following error: '{}'\n"
-                           "Attempting to finish cleanup".format(ex))
-            raise Exception("Failure(s) during cleanup") from ex
-        finally:
-            dirs = [os.path.join(self.conf.workspace, "tfout"),
-                self.tfjson_path]
-            for dir in dirs:
-                try: 
-                    self.utils.runshellcommand("rm -rf {}".format(dir))
-                except Exception as ex:
-                    logger.warning("Received the following error: '{}'\n"
-                                   "Attempting to finish cleanup".format(ex))
-
-    @timeout(600)
-    @step
-    def gather_logs(self):
-        logging_errors = False
+        self.tmp_files = [self.tfout_path,
+                          self.tfjson_path]
 
-        node_ips = {"master": self.get_nodes_ipaddrs("master"),
-                    "worker": self.get_nodes_ipaddrs("worker")}
-        logs = {"files": ["/var/run/cloud-init/status.json",
-                          "/var/log/cloud-init-output.log",
-                          "/var/log/cloud-init.log"],
-                "dirs": ["/var/log/pods"],
-                "services": ["kubelet"]}
+    def destroy(self, variables):
+        cmd = "destroy -auto-approve"
 
-        if not os.path.isdir(self.conf.log_dir):
-            os.mkdir(self.conf.log_dir)
-            logger.info(f"Created log dir {self.conf.log_dir}")
+        for var in variables:
+            cmd += f" -var {var}"
 
-        for node_type in node_ips:
-            for ip_address in node_ips[node_type]:
-                node_log_dir = self._create_node_log_dir(ip_address, node_type, self.conf.log_dir)
-                logging_error = self.utils.collect_remote_logs(ip_address, logs, node_log_dir)
+        self._run_terraform_command(cmd)
 
-                if logging_error:
-                    logging_errors = logging_error
-
-        platform_log_error = self._get_platform_logs()
-
-        if platform_log_error:
-            logging_errors = platform_log_error
-
-        return logging_errors
-
-    @step
-    def provision(self, num_master=-1, num_worker=-1):
+    def _provision_platform(self):
         """ Create and apply terraform plan"""
-        if num_master > -1 or num_worker > -1:
-            logger.warning("Overriding number of nodes")
-            if num_master > -1:
-                self.conf.master.count = num_master
-                logger.warning("   Masters:{} ".format(num_master))
-
-            if num_worker > -1:
-                self.conf.worker.count = num_worker
-                logger.warning("   Workers:{} ".format(num_worker))
-
+        exception = None
+        print("Init terraform")
```

Comment:
Should this be a logger call, or does it need printed?

Reply:
yep must of missed that one during the rebase

Reply:
I think cause it was removed I'll just remove it my next pr